### ISSUE TITLE
[Supporting the platform – Make Code - Extensions] Multiple link controls does not adapt high contrast(Black/White) theme under 'Extensions' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3692]
**User Experience**:
Visual impaired user will find difficulty to see the control or element if that control does not adopt the high contrast mode, as they won't be able to take on that controls and they will miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls appearing under "Extensions" page
8. Verify that Multiple link controls(Bluetooth, Devices, Radio-broadcast, Servo etc.) controls adapting high contrast theme or not.

**Actual Result:**
Multiple link controls(Bluetooth, Devices, Radio-broadcast, Servo etc.) does not adapt high contrast(Black/White) theme under 'Extensions' screen.

**Expected Result:**
Multiple link controls(Bluetooth, Devices, Radio-broadcast, Servo etc.) should adapt high contrast(Black/White) theme under 'Extensions' screen. It should look as 'Yellow' color in high contrast black theme and 'Blue' color in high contrast white theme.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.3.1; #Win10-Edge(Chromium)

![WCAG4 3 1_Extensions_HC Black Theme](https://user-images.githubusercontent.com/72871089/99036872-83267080-25a8-11eb-97e7-a512216cf2af.jpg)


![WCAG4 3 1_Extensions_HC White Theme](https://user-images.githubusercontent.com/72871089/99036879-86216100-25a8-11eb-85ec-76f254312f96.jpg)

**Comment by shakao** [comment]
these cards are not links, conceptually they aren't for page navigation--they trigger an action (adding an extension to your project) that then causes a page reload. closing, since they should not have the link color

**Comment by jwunderl** [comment]
(👍, re this I fixed their roles in https://github.com/microsoft/pxt/pull/8064/commits/6efddeb57d7ebd46e477523f51e3604048a32bd3 so they are also not listed as links anymore.)

**Comment by shakao** [comment]
oh good call, i'll update the roles for the language selector
